src/components/fx/hover.js

```diff
@@ -950,8 +950,13 @@ function createHoverText(hoverData, opts, gd) {
             var texts = getHoverLabelText(hoverData[j], true, hovermode, fullLayout, t0);
             var text = texts[0];
             var name = texts[1];
-            hoverData[j].text = name + ' : ' + text;
             hoverData[j].name = name;
+            if(name) {
```

Comment:
What happens if `name === 0`?

Reply:
Fixed in 36bc6b5dc2f27ed3c2b90066c0b1afc4199c692e by checking for empty string instead of truthy! Thanks @archmoj

Reply:
Resolved.